### PR TITLE
to fix NEXUS-10502...

### PR DESCRIPTION
**Pull request by wwannemacher**
the plugin needs to use a version of the nexus client lib that is fixed

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
   </scm>
 
   <properties>
-    <nexus.version>2.9.1-02</nexus.version>
+    <nexus.version>2.14.2-SNAPSHOT</nexus.version>
```
Comment (wwannemacher) [inline pom.xml]: no - the fix went in after the release... I guess that should be 2.14.3-SNAPSHOT - fixing now

**Comment by mosabua** [comment]
snapshots are not in the Central Repo so you will have to cut a release of RM first and then use the right version here...

**Comment by wwannemacher** [comment]
@mosabua - yeah, I wasn't sure how to make sure that this didn't get forgotten in the interim

**Comment by kazuki43zoo** [comment]
+1

**Pull request merged**